csharp/destinations/empty-template/ReadHelloWorld/Program.cs

```diff
@@ -1,49 +1,35 @@
-﻿using System;
-using System.Linq;
-using Quix.Sdk.Streaming;
-using Quix.Sdk.Streaming.Models;
-
-namespace ReadHelloWorld
-{
-    class Program
-    {
-        /// <summary>
-        /// Main will be invoked when you run the application
-        /// </summary>
-        static void Main()
-        {
-            // Create a client which holds generic details for creating input and output topics
-            var client = new QuixStreamingClient();
-
-            var inputTopicName = Environment.GetEnvironmentVariable("input");
-            
-            using var inputTopic = client.OpenInputTopic(inputTopicName);
-            
-            // Hook up events before initiating read to avoid losing out on any data
-            inputTopic.OnStreamReceived += (s, streamReader) =>
-            {
-                Console.WriteLine($"New stream read: {streamReader.StreamId}");
-                
-                var bufferConfiguration = new ParametersBufferConfiguration
-                {
-                    TimeSpanInMilliseconds = 1000,
-                };
-                
-                var buffer = streamReader.Parameters.CreateBuffer(bufferConfiguration);
-
-                buffer.OnRead += parameterData =>
-                {
-                    Console.WriteLine(
-                        $"ParameterA - {parameterData.Timestamps[0].Timestamp}: {parameterData.Timestamps.Average(a => a.Parameters["ParameterA"].NumericValue)}");
-                };
-            };
-
-            Console.WriteLine("Listening for streams");
-            
-            // Hook up to termination signal (for docker image) and CTRL-C and open streams
-            App.Run();
-
-            Console.WriteLine("Exiting");
-        }
-    }
-}
+﻿// Create a client which holds generic details for creating input and output topics
+using QuixStreams.Streaming;
+using QuixStreams.Streaming.Models;
+
+var client = new QuixStreamingClient();
+
+var topic = Environment.GetEnvironmentVariable("input");
+            
+using var consumer = client.GetTopicConsumer(topic);
+            
+// Hook up events before initiating read to avoid losing out on any data
+consumer.OnStreamReceived += (s, stream) =>
+{
+    Console.WriteLine($"New stream received: {stream.StreamId}");
+                
+    var bufferConfiguration = new TimeseriesBufferConfiguration
+    {
+        TimeSpanInMilliseconds = 1000,
+    };
+                
+    var buffer = stream.Timeseries.CreateBuffer(bufferConfiguration);
+
+    buffer.OnDataReleased += (sender, args) => {
+        
+        Console.WriteLine(

```

Comment:
Could you change this to just simple print of data object? Check if has any reasonable print, if not, just print something generic, like timestamp and number of parameters. This will fail on non-f1 topic...